src/devices/Mlx90614/Mlx90614.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers.Binary;
+using System.Device.I2c;
+using Iot.Units;
+
+namespace Iot.Device.Mlx90614
+{
+    /// <summary>
+    /// Infra Red Thermometer MLX90614
+    /// </summary>
+    public class Mlx90614 : IDisposable
```

Comment:
This should probably be a sealed class, which might help when looking that the x in the name is actually part of the name and not meaning that it is a base class for a device family.